plugins/TunePMuonProducer.cc

```diff
@@ -0,0 +1,47 @@
+#include "DataFormats/PatCandidates/interface/Muon.h"
```

Comment:
I'm puzzled. Why do you need a producer, whilst it seems that you are only selecting some muons in the muon collection? Cannot you use a PATmuonSelector for that? or add the ```mu.tunePMuonBestTrack().isAvailable()``` to the selection in the skim?